src/android/io/branch/BranchSDK.java

```diff
@@ -1127,7 +1138,9 @@ public void run() {
                     } else if (this.action.equals("logout")) {
                         logout(this.callbackContext);
                     } else if (this.action.equals("loadRewards")) {
-                        loadRewards(this.callbackContext);
+                        if (this.args.length() == 1) {
+                            loadRewards(this.callbackContext, this.args.getString(0));   
```

Comment:
@aaustin ,  added overload method for `loadRewards` optional parameter bucket string.